### PR TITLE
Package with assets for the Material study app Rally

### PR DESCRIPTION
**Pull request by perclasson**
Adding files for rally_assets package.

### CODE REVIEW

**Comment by perclasson** [comment]
Will publish the package after it has been merged to master.

From running `flutter pub publish --dry-run`:
    
    Publishing rally_assets 1.0.0 to https://pub.dartlang.org:
    |-- .gitignore
    |-- .metadata
    |-- AUTHORS
    |-- CODE_OF_CONDUCT.md
    |-- LICENSE
    |-- lib
    |   |-- Eczar-Bold.ttf
    |   |-- Eczar-Regular.ttf
    |   |-- Eczar-SemiBold.ttf
    |   |-- RobotoCondensed-Bold.ttf
    |   |-- RobotoCondensed-Light.ttf
    |   |-- RobotoCondensed-Regular.ttf
    |   |-- logo.png
    |   '-- thumb.png
    '-- pubspec.yaml
    Suggestions:
    * Please add a README.md file that describes your package.
    
    Package has 1 warning.
    pub finished with exit code 65

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,98 @@
+Copyright (c) 2014, Vaibhav Singh (design) and Rosetta Type Foundry s.r.o. (post-production).
```
Comment (rami-a) [inline LICENSE]: Is this file supposed to be here? looks like there is another license file under `rally_assets/LICENSE`

Comment (perclasson) [reply LICENSE]: I believe that we want to keep one LICENSE file for the whole repo in the root directory that includes all licenses and the reason we have a separate in the rally_assets directory, as well as the shrine_images directory, is to make sure that it gets added whenever someone is downloading the package.

Comment (rami-a) [reply LICENSE]: ah gotcha, I originally thought this was a new file and didn't realize that it was amending the existing one. Cool

File: rally_assets/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
```
Comment (clocksmith) [inline rally_assets/pubspec.yaml]: nit: add newline at end of file

Comment (perclasson) [reply rally_assets/pubspec.yaml]: Done!

**Pull request merged**